package-system/AwsIotDeviceSdkCpp/readme.txt

```diff
@@ -4,12 +4,12 @@ A few things to be aware of:
 
 1. This sdk requires an extremely short build path to build (on Windows only), such as 'd:\aws' or 'd:\temp'.  Otherwise it will fail to build due to excessively long file names.  
 
-2. As of v1.12.2, the build scripts for the aws-c-common and aws-checksums submodules need to be patched to build their libraries correctly into Debug and Release subdirectories.  Without the patches, the base iot libraries will build into the subdirectories, but the aws-c-common libraries will build into a root 'lib' directory and overwrite each other when the different configurations are built.  These patches have been submitted as PRs, but have not yet been accepted or integrated back into aws-iot-device-sdk-cpp-v2.
+2. As of v1.15.2, the build scripts for the aws-c-common and aws-checksums submodules need to be patched to build their libraries correctly into Debug and Release subdirectories.  Without the patches, the base iot libraries will build into the subdirectories, but the aws-c-common libraries will build into a root 'lib' directory and overwrite each other when the different configurations are built.  These patches have been submitted as PRs, but have not yet been accepted or integrated back into aws-iot-device-sdk-cpp-v2.
 	Submitted pull requests:
 	https://github.com/awslabs/aws-checksums/pull/47
 	https://github.com/awslabs/aws-c-common/pull/792
 
-3. The iOS version doesn't currently build, it has the following error:
```

Comment:
Have a list at the top to show supported platform and not supported platform?
What is the point to show the error here? If there is no guide how to build IOS, just remove this section

Reply:
I updated the file to list the supported/unsupported platforms at the top. Also simplified this section to remove the detailed errors but keep the section here for debugging purpose so that we can check whether this issue has been fixed or still blocked when we start investigating the mobile platforms.